### PR TITLE
[AIRFLOW-2678] Fix db schema unit test to remove checking fab models

### PR DESCRIPTION
**Pull request by feng-tao**
Make sure you have checked _all_ steps below.

### JIRA
- [x] My PR addresses the following [Airflow JIRA](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
    - https://issues.apache.org/jira/browse/AIRFLOW-2678
    - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a JIRA issue.

### Description
- [x] Here are some details about my PR, including screenshots of any UI changes:
Currently airflow doesn't include any model file for FAB. If fab is enabled, this unit test will fail(complaining fab model is removed)

IMO, since all the FAB models are external, we shouldn't check those models.

### Tests
- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits
- [ ] My commits all reference JIRA issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

### Documentation
- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
    - When adding new operators/hooks/sensors, the autoclass documentation generation needs to be added.

### Code Quality
- [x] Passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

### CODE REVIEW

**Comment by feng-tao** [comment]
PTAL @jgao54 @seelmann

**Comment by seelmann** [comment]
I run the tests manually with `RBAC` enabled and they passed so I assumed the merging of the models worked. But I agree that they can be removed as there are no migration scripts for them.

**Review by seelmann — APPROVED** [review]

**Comment by feng-tao** [comment]
@seelmann , I suspect when you run the test, the RBAC tables haven't been created. I encounter this issue both in jenkins and local test.

**Comment by feng-tao** [comment]
hey Joy @jgao54 , could you take a look? thanks.

File: tests/utils/test_db.py [hunk 1]
```diff
@@ -35,9 +34,6 @@ def test_database_schema_and_sqlalchemy_model_are_in_sync(self):
         all_meta_data = MetaData()
         for (table_name, table) in airflow_base.metadata.tables.items():
             all_meta_data._add_table(table_name, table.schema, table)
-        if RBAC:
```
Comment (jgao54) [inline tests/utils/test_db.py:38]: nit: can we also remove the comment `# combine Airflow and Flask-AppBuilder (if rbac enabled) models`

**Comment by jgao54** [comment]
sounds good, agree it is not necessary right now.

File: tests/utils/test_db.py (hunk 1 above)
Comment (feng-tao) [reply tests/utils/test_db.py:38]: thanks. I update the pr and remove that line.

**Comment by feng-tao** [comment]
thanks @jgao54 , pr updated.

**Comment by jgao54** [comment]
thanks! lgtm :)

**Pull request merged**